h2o-r/tests/testdir_munging/runit_pivot.R

```diff
@@ -0,0 +1,21 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+test.pivot <- function(){
+    print("step1")
+    df = h2o.createFrame(rows = 1000,
+                         cols=3,
+                         factors=10,
+                         integer_fraction=1.0/3,
+                         categorical_fraction=1.0/3,
+                         missing_fraction=0.0,
+                         seed=123)
+    df$C3 = h2o.abs(df$C3)
+    head(df)
+    df2 = h2o.pivot(df,index="C3",column="C2",value="C1")
+    head(df2)
+    expect_equal(nrow(df2),101)
+    expect_equal(ncol(df2),11)
```

Comment:
I think it would be bad to reduplicate the same tests in both Python and R, because of redundancy and bogging down of the test system. I just have the R test here to test the client.  And the Python tests works on more of the bigger test sets and corner cases